static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx

```diff
@@ -410,7 +411,7 @@ export function ServerSideSampling({project}: Props) {
             'These settings can only be edited by users with the organization owner, manager, or admin role.'
           )}
         />
-        {!!rules.length && (
+        {!!rules.length && !fetchingRecommendedSdkUpgrades && (
```

Comment:
Is this check needed? Won't `SamplingSDKAlert` return null if there are no SDK upgrades while loading?

Reply:
yes, that is true, but why render the component if we have this fetching now? 🤷‍♀️